### ISSUE TITLE
Client stuck if server closed while client is in splash screen 

### ISSUE THREAD

**Issue by qqii** [CosyClub/ProjectTempo#177]
This occurred during the testing day.

**Comment by AnthonyWharton** [comment]
Nope, the client will time out after 60 seconds. I agree it's not good, but I think the issue here should be more to do with how the splash screen is implemented (not in the main loop, and then calls the connection code on "any key to continue", which causes a delay between the key press and entering the game.

**Comment by qqii** [comment]
Just going to test this now.
